src/redux/books/books.js

```diff
@@ -3,28 +3,75 @@
 import * as actions from '../actionTypes';
 import initialData from './initialData';
 
-let lastId = 0;
+const url = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/FsmSXjrNKGR3sQq9YSmL/books';
+
+export const getBooks = () => (dispatch) => {
+  dispatch(actions.getBooksLoading());
+  fetch(url)
+    .then((response) => response.json())
+    .then((data) => {
+      const formattedBooks = [];
+      Object.keys(data).forEach((key) => {
+        if (key) {
+          formattedBooks.push({ ...data[key][0], item_id: key });
+        }
+      });
+      dispatch(actions.getBooksSuccess(formattedBooks));
+    })
+    .catch((err) => {
+      dispatch(actions.getBooksFailure(err.message));
+    });
+};
+
+export const removeBook = (id) => (dispatch) => {
+  fetch(`${url}/${id}`, {
+    method: 'DELETE',
+  })
+    .then(() => {
+      dispatch(getBooks());
+    });
+};
```

Comment:
- [ ] Good job so far 👍 I can add and delete books but there are two hard-coded books that I can't seem to delete. Please adjust the functionality so that I can delete any book.
- [ ] Also when I add a book, I can't see the title of the book. Kindly fix that.
- [ ] Kindly also fix the following console error 👍 
-
![Annotation 2022-05-06 205146](https://user-images.githubusercontent.com/85113567/167208880-e55bf8d6-8a49-42ac-9d0b-567123ed41c8.jpg)
- Console error:
![Annotation 2022-05-06 205042](https://user-images.githubusercontent.com/85113567/167208908-f402af68-f568-4016-8de1-8eb3c94bb305.jpg)